packages/cli/generators/extension/templates/src/types.ts.ejs

```diff
@@ -0,0 +1,7 @@
+export interface <%= project.optionsInterface %> {
```

Comment:
Let's add some comments here to explain the purpose of `<Extension>Options` and describe specific properties should be added into the empty body.